### PR TITLE
Add util.ssh module for TemporarySSHKey

### PR DESCRIPTION
**Pull request by mbarnes**
Needed `TemporarySSHKey`, and the `commissaire` module seemed the best fit.

I did a bit of module renaming.  `commissaire.config` is now `commissaire.util.config`, and I added `TemporarySSHKey` in `commissaire.util.ssh`.  Pretty much like what we had before.  If that's not cool I can change it around.

### CODE REVIEW

File: test-requirements.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 tox
 behave
 requests
+mock
```
Comment (ashcrow) [inline test-requirements.txt]: `mock` is now part of `unittest` in Python 3.

File: src/commissaire/utils/ssh.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Copyright (C) 2016  Red Hat, Inc
```
Comment (ashcrow) [inline src/commissaire/utils/ssh.py]: Accidental dupe file?

**Review by ashcrow — CHANGES_REQUESTED** [review]

File: src/commissaire/utils/ssh.py (hunk 2 above)
Comment (mbarnes) [reply src/commissaire/utils/ssh.py]: Yes.  Whoops!  :confounded:

**Comment by mbarnes** [comment]
:arrow_up:

**Review by ashcrow — APPROVED** [review]

**Comment by ashcrow** [comment]
Manually merged.

**Pull request closed without merging**